src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+using Microsoft.Maui.Controls;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// DrawingView Platform Control
+/// </summary>
+public partial class MauiDrawingView
```

Comment:
After this second review, I'm thinking if this implementation shouldn't be on `Controls` project

Reply:
in that case, we should move Alerts and Popup to the `Controls` project as well